NomadicNibbler/foodtruck-fe
# Feature/form error handling

## What does this PR do?

- Adds error handling for zip codes
- Adds error handling for duplicate usernames
- Adds error handling for incorrect usernames
- Adds error messages for these things
- Redirects users to appropriate pages when these errors happen
- Adds testing for these features. plus a few other things

## How should it be tested?

- login with a fake username, and you should be redirected to login screen, and see an error message
- create a new user with a username you know already exists, and you should see an error message after being redirected back to the new user form page.

## What do the future iterations hold?

- Fix errors.

## Linked Issues
- #35
- #36
- #37 
- #
